src/commonmark/en/content/developer/web-api.md

```diff
@@ -788,13 +782,13 @@ locales).
 <!--DHIS2-SECTION-ID:webapi_api_versions-->
 
 The Web API is versioned starting from DHIS 2.25. The API versioning
-follows the DHIS 2 major version numbering. As an example, the API
-version for DHIS 2.25 is *25*.
+follows the DHIS2 major version numbering. As an example, the API
+version for DHIS 2.25 is `25`.
 
 You can access a specific API version by including the version number
-after the */api* component, as an example like this:
+after the `/api` component, as an example like this:
 
-    /api/26/dataElements
+    /api/dataElements
```

Comment:
The text above suggests that this should be an example _with_ a version number.

Reply:
Fixed.